platform/kubernetes.go

```diff
@@ -26,9 +30,105 @@ import (
 	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
 	"k8s.io/client-go/rest"
 	"k8s.io/client-go/tools/clientcmd"
+	clientcmdapi "k8s.io/client-go/tools/clientcmd/api"
 	//This is required for gcp auth provider scope
 )
 
+// Setcontext update the file ~/.kube/config with the context associate with teh cluster.
+func UnSetK8SContext(GRVCluster configuration.ProviderCluster) error {
```

Comment:
gravitywell just uses GOOGLE_APPLICATION_CREDENTIALS to authenticate with kubernetes, so I do not think this function is needed :thinking: